### PR TITLE
Remove international phone validation library

### PR DESCRIPTION
**Pull request by ArayB**
The library was preventing valid numbers for Brazilian mobile phones
being added.
The library has not been updated in 2 years so unlikely to be fixed.

Removing phone number validation as trying to validate phone number
format is incredibly difficult with all the different international
formats possible.

### CODE REVIEW

File: web/public/js/directives/cd-start-dojo/champion/template.dust [hunk 1]
```diff
@@ -50,11 +50,9 @@
     <h3>{@i18n key="Contact details to help us verify your Dojo"/}</h3>
     <div class="form-group">
       <label for="phone">{@i18n key="Phone number"/}</label>
-      <input type="tel" name="phone" id="phone" autocomplete="off" class="form-control"
-        utils-script="/components/intl-tel-input/lib/libphonenumber/build/utils.js"
-         required cd-phone-validator international-phone-number ng-model="$ctrl.champion.phone">
+      <input type="tel" name="phone" id="phone" class="form-control" required ng-model="$ctrl.champion.phone">
         <div ng-messages="$ctrl.championForm.phone.$error" ng-show="$ctrl.championForm.phone.$touched && !$ctrl.championForm.phone.$valid">
-          <label class="control-label has-error validationMessage">{@i18n key="Telephone number is not valid"/}</label>
```
Comment (Wardormeur) [inline web/public/js/directives/cd-start-dojo/champion/template.dust]: That means cp-translation should be updated. The key for that translation changed :)

**Comment by Wardormeur** [comment]
Pretty sure that injection should be removed as well https://github.com/CoderDojo/cp-zen-platform/blob/master/web/public/js/cp-zen-platform.js#L28

File: web/public/js/directives/cd-start-dojo/champion/template.dust (hunk 1 above)
Comment (ArayB) [reply web/public/js/directives/cd-start-dojo/champion/template.dust]: The "Telephone Number is empty" key already exists in cp-translations, assuming the `strings/en_US/messages.po` is the right place to check

Comment (Wardormeur) [reply web/public/js/directives/cd-start-dojo/champion/template.dust]: Apologies, all good then !

**Pull request merged**